Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionControllerConfiguration.mm

```diff
@@ -59,7 +59,9 @@ + (instancetype)configurationWithIdentifier:(NSUUID *)identifier
 {
     NSParameterAssert(identifier);
 
-    return WebKit::WebExtensionControllerConfiguration::create(identifier)->wrapper();
+    auto uuid = UUID::fromNSUUID(identifier);
+    ASSERT(uuid);
```

Comment:
Probably this should be RELEASE_ASSERT, since the consequence may effect data on disk.